src/GraphQL/Types/Scalars/IdGraphType.cs

```diff
@@ -52,7 +50,7 @@ public IdGraphType()
             string _ => value,
             int _ => value,
             long _ => value,
-            Guid _ => value,
+            Guid _ => value, //TODO: wtf???
```

Comment:
@Shane32  ping

Reply:
All of the scalars allow parsing native types from variables.  So does IdGraphType.  Is there a question?

Reply:
Guid, int, long, and string are probably the most commonly used types with IdGraphType.

Reply:
ok